Libraries/CommonFunctions.psm1

```diff
@@ -1028,7 +1028,7 @@ function Get-VMFeatureSupportStatus {
 
 	Write-Output "yes" | .\Tools\plink.exe -C -pw $Password -P $SSHPort $Username@$Ipv4 'exit 0'
 	$currentKernel = Write-Output "yes" | .\Tools\plink.exe -C -pw $Password -P $SSHPort $Username@$Ipv4  "uname -r"
-	if ( $LASTEXITCODE -eq $false) {
+	if ( $LASTEXITCODE -ne 0 ) {
 		Write-LogInfo "Warning: Could not get kernel version".
```

Comment:
if $currentKernel is $null, https://github.com/LIS/LISAv2/blob/a87d97a2f43e7fb2a4b4b2de7afee6bd05ae69c0/Libraries/CommonFunctions.psm1#L1035 this line will throw exception?

Reply:
@LiliDeng thank you so much for your help to review. 
Yes, it will throw exception, following list original test result without this PR. Thank you.

04/08/2019 09:52:04 : [INFO ] TestParams : ';CycleCount=20'
04/08/2019 09:52:25 : [ERROR] EXCEPTION: You cannot call a method on a null-valued expression.
04/08/2019 09:52:25 : [ERROR] Source: Line 1035 in script C:\Users\xuli\workspace\repo\LISAv2-1\Libraries\CommonFunction
s.psm1.
04/08/2019 09:52:25 : [INFO ] Trying to connect to deployed VMs.

[LISAv2 Test Results Summary]
Test Run On           : 04/08/2019 09:49:40
VHD Under Test        : C:\RHEL-8.0.0-20190213.0-x86_64-GEN1-A.vhdx
Total Test Cases      : 1 (0 Passed, 1 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:4

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 KVP                  KVP-TEST-RESCIND                                                                  FAIL                 2.48